src/invariant.ts

```diff
@@ -253,8 +249,7 @@ export class Invariant {
           xToY,
           amount,
           byAmountIn,
-          sqrtPriceLimit,
-          approvedAmount
+          sqrtPriceLimit
```

Comment:
If `approvedAmount` has been removed from `TxScript` is it still required to pass it as parameter?

Reply:
Currently yes, we need to pass both the amount we want to receive and the approved one. In TxScripts they are passed in braces syntax.